packages/editor-ui/src/api/cloudPlans.ts

```diff
@@ -0,0 +1,34 @@
+import type { CloudPlanData, IRestApiContext } from '@/Interface';
+
+export function getCurrentPlan(
+	context: IRestApiContext,
+	cloudUserId: string,
+): Promise<CloudPlanData> {
+	return Promise.resolve({
+		planSpec: {
```

Comment:
nitpick - maybe we should rename the key to plan from planSpec.. since expirationDate is a computed property.. so it's not a spec anymore

Reply:
or else move `expirationDate` under `usage`.. or else flatten all the objects here..